tools/serverpod_cli/bin/shared/environment.dart

```diff
@@ -6,8 +6,11 @@ String serverpodHome = '';
 
 bool loadEnvironmentVars() {
   if (!productionMode) {
-    var home = Platform.environment['SERVERPOD_HOME'];
```

Comment:
The SERVERPOD_HOME is not the same as the users home directory. It should point to the serverpod directory (as cloned from Github). This change will break things for local development if the SERVERPOD_HOME isn't defined.

Reply:
Where are we assigning it a value? Any env configs or something? As we are not assigning it any value, it will be null forever.